src/ORM/Table.php

```diff
@@ -844,6 +845,10 @@ public function behaviors(): BehaviorRegistry
      *
      * @param string $name The behavior alias to get from the registry.
      * @return \Cake\ORM\Behavior
+     * @template TName of key-of<TBehaviors>
+     * @phpstan-param TName $name The behavior alias to get from the registry.
+     * @phpstan-return TBehaviors[TName]
+     * @psalm-return \Cake\ORM\Behavior
```

Comment:
psalm-return looks the same as return.

Reply:
Yes, but see the commits above. If not needed, psalm goes all ballistic on us.

Reply:
This is needed because psalm reads `@phpstan-` prefixed annotations too (and vice versa) and it doesn't understand `TBehaviors[TName]`, hence `@psalm-return` is also required to override the `@phpstan-return`.